components/PresaleHeader.tsx

```diff
@@ -11,82 +11,72 @@ import moment from 'moment'
 export function PresaleHeader({ className }: SectionProps) {
 
   return (
-    <div
+    <div className={classNames("absolute w-full bg-blue flex justify-center", className)} style={{ height: '5rem' }}>
+      <div
       className={classNames(
-        'desktop:container w-full grid grid-cols-3 items-center justify-items-start bg-blue text-white gap-x-2 font-graphik laptop:absolute laptop:h-28 desktop:px-14',
-        'grid-cols-teasingPresaleHeader laptop:grid-cols-presaleHeader ',
-        className
+        'desktop:container grid grid-cols-3 items-center justify-items-start bg-blue text-white gap-x-2 font-graphik laptop:h-28 desktop:px-14',
```

Comment:
Revert to the previous version with only one `<div>` and remove `desktop:container`. Then add your changes, and make sure to only use one `<div>`